core/src/test/scala/unit/kafka/log/LogCleanerLagIntegrationTest.scala

```diff
@@ -44,18 +44,19 @@ class LogCleanerLagIntegrationTest(compressionCodecName: String) extends Logging
 
   val time = new MockTime(1400000000000L, 1000L)  // Tue May 13 16:53:20 UTC 2014 for `currentTimeMs`
   val cleanerBackOffMs = 200L
-  val segmentSize = 100
+  val segmentSize = 512
```

Comment:
Seems reasonable to me. cc @junrao in case he knows a reason why we should keep the segment size at `100`.